src/metadata_utils.jl

```diff
@@ -93,43 +88,46 @@ metadata_model(KNNRegressor,
     input_scitype=MLJModelInterface.Table(MLJModelInterface.Continuous),
     target_scitype=AbstractVector{MLJModelInterface.Continuous},
     supports_weights=true,
-    docstring="K-Nearest Neighbors classifier: ...",
     load_path="NearestNeighbors.KNNRegressor")
 ```
 """
 function metadata_model(
     T;
     # aliases:
-    input=Unknown,
-    target=Unknown,
-    output=Unknown,
-    weights::Bool=false,
-    descr::String="",
-    path::String="",
+    input=nothing,
+    target=nothing,
+    output=nothing,
+    weights::Union{Nothing,Bool}=nothing,
+    class_weights::Union{Nothing,Bool}=nothing,
+    descr::Union{Nothing,String}=nothing,
+    path::Union{Nothing,String}=nothing,
 
     # preferred names, corresponding to trait names:
     input_scitype=input,
     target_scitype=target,
     output_scitype=output,
-    supports_weights=weights,
-    docstring=descr,
-    load_path=path,
+    supports_weights::Union{Nothing,Bool}=weights,
+    supports_class_weights::Union{Nothing,Bool}=weights,
```

Comment:
shouldn't this be 
```suggestion
    supports_class_weights::Union{Nothing,Bool}=class_weights,
```

Reply:
Yikes! I thought I caught that one already. Will fix and improve testing.